packages/codec/lib/compilations/utils.ts

```diff
@@ -120,9 +129,9 @@ export function shimContracts(
     let sourceObject: Source = {
       sourcePath,
       source,
-      ast: <Ast.AstNode>ast,
+      ast: <AstNode>ast,
       compiler,
-      language: inferLanguage(<Ast.AstNode>ast, compiler, sourcePath)
+      language: inferLanguage(<AstNode>ast, compiler, sourcePath)
```

Comment:
Because going by the contract schema, AST nodes are an `any`, or maybe not an `any` but something similarly broad.  So we have to coerce for things that expect an actual AST node.

Reply:
gotcha